src/microbe_stage/systems/EngulfedDigestionSystem.cs

```diff
@@ -107,6 +103,17 @@ protected override void Update(float delta, in Entity entity)
                 if (engulfable.PhagocytosisStep != PhagocytosisPhase.Ingested)
                     continue;
 
+                // Disable engulfed entities from absorbing compounds
+                if (engulfedObject.Has<CompoundAbsorber>())
```

Comment:
This needs to be reset back to normal when ejected. So here's my suggestion put this (and the below code) to the on engulfed method that sets up various things like disabling engulfed object physics. It makes a lot more sense to be in there in my opinion.